docs/dev/error-messages.md

```diff
@@ -241,6 +241,28 @@ This isn't always restricted to the tests, either; an app under test which consu
 
 Break out your long tests into shorter tests and/or make sure that your tests are not filling up a lot of disk space on the VM.
 
+
+### Unable to Find Device Within 15 Minutes
+
+**Description**
+
+This timeout occurs most frequently when you include a specific device in your test case and that device isn’t available within 15 minutes.
+
+**Cause(s)**
+
+Our public device pool is available to all subscribed Sauce users and (as you might imagine), some devices are more popular than others. We have over 280 device configurations and thousands of devices hosted in our data center, but sometimes a test queue will build up for the most popular devices.
```

Comment:
@aoleszyn 

I know this is a copy, but this message is not specifically for the public cloud. It's for both, so let's remove `public`